### PR TITLE
synched with pib-beta-fleet

### PR DESCRIPTION
**Pull request by bartr**
## Purpose of PR

- synched with pib-beta-fleet
- use pib-service to deploy heartbeat

## Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [x] CI-CD changes
- [ ] GitHub Template changes

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/pib-cli/issues/657

### CODE REVIEW

**Review by kev-ms — APPROVED** [review]
Do we need k3d since we're not runnnig innerloop as part of this repo?

**Comment by bartr** [comment]
"we" need the .devcontainer/k3d.yaml file for dev and test

I'm OK leaving in beta-fleet and not pib-gitops - I'll delete it from the PR

File: .devcontainer/on-create.sh [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
+# update these values for DNS / SSL support
+
+# set env vars
+#export PIB_MI=yourAzureManagedIdentity
+#export PIB_DNS_RG=yourDNSResourceGroup
+#export PIB_SSL=yourDomain
 
-export PATH="$PATH:$PIB_BASE/cli"
 export GOPATH="$HOME/go"
+export PATH="$PATH:$PIB_BASE/cli"
```
Comment (aflinchb) [inline .devcontainer/on-create.sh:17]: Is PIB_BASE set at this point?

**Review by aflinchb — APPROVED** [review]
Fixed 2 tiny typos

File: .devcontainer/on-create.sh (hunk 1 above)
Comment (bartr) [reply .devcontainer/on-create.sh:17]: good catch - that won't work ...

I'll fix it

**Pull request merged**